Gemfile

```diff
@@ -2,7 +2,6 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in experiments.gemspec
 gemspec
-gem "rubysl", platform: :rbx
```

Comment:
This Gem seems like it is needed only for Ruby <2.0 and was probably not installed on CI before.